app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -393,6 +401,28 @@ def volume_description(volume)
       return volume.description
     end
 
+    def parse_backup(backup)
+      uid = backup['id']
+      new_result = {
+        :ems_ref               => uid,
+        :type                  => "ManageIQ::Providers::Openstack::CloudManager::CloudVolumeBackup",
+        # Supporting both Cinder v1 and Cinder v2
+        :name                  => backup['display_name'] || backup['name'],
+        :status                => backup['status'],
+        :creation_time         => backup['created_at'],
+        # Supporting both Cinder v1 and Cinder v2
+        :description           => backup['display_description'] || backup['description'],
+        :size                  => backup['size'].to_i.gigabytes,
+        :object_count          => backup['object_count'].to_i,
+        :is_incremental        => backup['is_incremental'],
+        :has_dependent_backups => backup['has_dependent_backups'],
+        :availability_zone     => @data_index.fetch_path(:availability_zones,
+                                                         "volume-" + backup['availability_zone'] || "null_az"),
```

Comment:
What is purpose of AZ name ```"volume-" + backup['availability_zone']```? It looks to me it might cause https://bugzilla.redhat.com/show_bug.cgi?id=1377393